pkg/runctl/run_manager.go

```diff
@@ -381,7 +401,37 @@ func (c *runManager) setupNetworkPolicyFromConfig(runNamespace string) error {
 	return nil
 }
 
-func (c *runManager) createTektonTaskRun(pipelineRun k8s.PipelineRun) error {
+func (c *runManager) volumesWithSaSecret(ctx *runContext) []corev1api.Volume {
+	var mode int32 = 420
```

Comment:
Bettter write as octal number:
```suggestion
	var mode int32 = 0644
```